<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin_Window</class>
 <widget class="QMainWindow" name="Admin_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>747</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>371</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame1{
background-color: qlineargradient(spread:pad, x1:0.920727, y1:0.881, x2:0.693, y2:0.795818, stop:0.107955 rgba(0, 0, 0, 255), stop:0.642045 rgba(255, 255, 255, 255));
border-radius:20px}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;微软雅黑&quot;;
font: 75 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>管理员界面</string>
     </property>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>371</width>
       <height>161</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	background-color:rgba(255,255,255,150);
	border:none;
	border-bottom:1px solid black

}
QPushButton{
	bordeo-radius:8px
}
QPushButton:pressed{
	padding-top:5px;
	padding-left:5px;
}</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_3">
      <widget class="QPushButton" name="pushButton_search">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>110</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_search">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>80</y>
         <width>251</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>输入想要查询的人员</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>60</y>
         <width>211</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>（用户/值班/安全设施）</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_exit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>400</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>541</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame{
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0.341, y2:0.489, stop:0 rgba(0, 0, 0, 255), stop:0.375 rgba(255, 255, 255, 255));
border-radius:25px}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>541</width>
       <height>421</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#stackedWidget{
background-color: qlineargradient(spread:pad, x1:0.966, y1:0.914773, x2:0.534091, y2:0.716, stop:0 rgba(0, 0, 0, 255), stop:0.323864 rgba(255, 255, 255, 255));}</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page"/>
     <widget class="QWidget" name="page_2">
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>23</x>
         <y>6</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>查询结果：</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>551</width>
         <height>291</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="plainText">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_111">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>380</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton:pressed{
	padding-top:5px;
	padding-left:5px;
}</string>
       </property>
       <property name="text">
        <string>归还</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_222">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>380</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton:pressed{
	padding-top:5px;
	padding-left:5px;
}</string>
       </property>
       <property name="text">
        <string>未归还</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_black">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>380</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton:pressed{
	padding-top:5px;
	padding-left:5px;
}</string>
       </property>
       <property name="text">
        <string>关小黑屋</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_white">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>380</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton:pressed{
	padding-top:5px;
	padding-left:5px;
}</string>
       </property>
       <property name="text">
        <string>移除小黑屋</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_object">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>330</y>
         <width>171</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>320</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>输入用户id来进行以下操作：</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
